.github/workflows/deploy-website.yml

```
@@ -0,0 +1,36 @@
+name: Deploy to NuxtHub
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)